packages/gasket-plugin-morgan/lib/index.js

```diff
@@ -0,0 +1,24 @@
+const morgan = require('morgan');
+const split = require('split');
+const { name } = require('../package');
+
+// set log configuration in gasket.config.js, under `morgan` key
+// configuration options: http://expressjs.com/en/resources/middleware/morgan.html
+module.exports = {
+  name,
+  dependencies: ['@gasket/plugin-express', '@gasket/plugin-log'],
```

Comment:
This should work for any plugin that executes the `middleware` lifecycle, such as [@gasket/plugin-fastify](https://github.com/godaddy/gasket/tree/main/packages/gasket-plugin-fastify). Whereas log is necessary dependency for it to work.

```suggestion
  dependencies: ['@gasket/plugin-log'],
```